src/app/components/Textarea/Textarea.tsx

```diff
@@ -23,8 +22,11 @@ export default function Textarea({
         className={styles.textarea}
         maxLength={40}
         value={value}
-        onChange={(event) => onChange(event.target.value)}
+        onChange={(event) => {
+          event.preventDefault();
```

Comment:
I don´t know if you need it cause your form on your Calendarpage, where your component is in, have all ready an event.preventDefault()